### DISCUSSION TITLE
How to define a reflective BC？

### DISCUSSION THREAD

**Discussion by lipxisi** [idaholab/moose#23712] [Q&A General] [answered]
Hollow, I`m a new MOOSE user.  How to define a reflective BC in the 1/4 core. Are the reflective BC described in the [Problem] or in [BC]? 
Thanks!

**Comment by GiudGiud** [comment, marked as answer]
Hello

It should be in the BC block
Which physics are you modeling? Often reflective is the same as the default no BC

Guiillaume

**Reply by lipxisi** [reply]
Thanks your answer. I want to build a 1/4 core model, the neutron flux BC in top and left are zero, and in right and bottom are mirror symmetry BC. So do I need to specify this symmetry problem in the [problem] block of the input file or specify symmetric boundary conditions in the [BC]? And how to realize the definition of mirror symmetry BC. I can't find help in the relevant instructions on the official website.
Thanks again.

**Reply by GiudGiud** [reply]
For a classic diffusion problem, there's no need to specify a BC to get a reflective BC. The default is a 0 gradient (eg 0 current), which is the reflective BC. 

Are you not getting a 0 gradient at the boundary with no BC?
